hedera-node/src/main/java/com/hedera/services/store/contracts/ContractsStore.java

```diff
@@ -86,6 +104,30 @@ public void prepareAccountCreation(AccountID sponsor, AccountID target, HederaAc
 		provisionalAccountCreations.put(target, new AbstractMap.SimpleEntry<>(sponsor, customizer));
 	}
 
+	public com.hedera.services.store.models.Account loadContract(Id id) {
+		final var key = new MerkleEntityId(id.getShard(), id.getRealm(), id.getNum());
+		final var merkleAccount = contracts.get().get(key);
+
+		validateUsable(merkleAccount);
+
+		final var account = new com.hedera.services.store.models.Account(id);
+		account.setExpiry(merkleAccount.getExpiry());
+		account.initBalance(merkleAccount.getBalance());
+		account.setAssociatedTokens(merkleAccount.tokens().getIds().copy());
+
+		return account;
+	}
+
+	public void persistContractResult(TransactionProcessingResult result) {
```

Comment:
- We are persisting changes in the `commit` method. I think that it is better to rename this method to `externaliseResult` and inside this method to simply call the transaction record service with the `TransactionProcessingResult`. The Transaction record service will create the protobuf file and set it.
- Add javadoc